test/t/unit/test_unit_command_offset.py

```diff
@@ -30,9 +31,12 @@ def functions(self, bash):
             'complete -W %s "${COMP_WORDS[0]}"' % quote(join(self.wordlist)),
             'COMPREPLY=(dummy); complete -r "${COMP_WORDS[0]}"',
             "COMPREPLY+=(${#COMPREPLY[@]})",
+            'complete -C ./completer "${COMP_WORDS[0]}"',
```

Comment:
This array is also for 124. I think the following line should be added at the end of `functions(self,bash)` instead:

```bash
        assert_bash_exec(bash, "complete -C ./completer cmd6")
```

Reply:
Thank you!

You have put the above line in `test_3` instead of in `functions`. I'm interested in if there is a reason that this line is put directly in `test_3`. If we put it in `test_3`, it will be executed every time for each `(cmd,expected_result)` pair. But I think this common setup can be executed just once, which is one of the reasons that one defines completion settings with `@pytest.fixture`.

<!-- Could you move the line to `functions`? Or, if you think this setting shouldn't be put in `functions`, please also remove the argument `functions` from `test_3` since `test_3` doesn't depend on `functions` in that case. -->

Reply:
Whoops! I got confused and put it in the wrong place. Fixing now.